.github/workflows/test.yml

```diff
@@ -60,13 +60,19 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, windows-latest]
+        os: [ubuntu-latest]
         python-version: ["3.9", "3.10", "3.11", "3.12", "3.13", "3.14", pypy-3.11]
         include:
         - os: macos-latest
           python-version: "3.9"
         - os: macos-latest
           python-version: "3.13"
+        - os: windows-latest
+          python-version: "3.9"
+        - os: windows-latest
+          python-version: "3.13"
+        - os: windows-latest
+          python-version: pypy-3.11
```

Comment:
I may veto this as it has caused too many issues in the past.

Reply:
Yeah I just wanted to see if I can trigger the bug more easily on PyPy.

Reply:
Anyway it seems the test suite won't run if there is no file change?

Reply:
You can omit that from the test workflow for the time being. Just put it back when this is ready for review.